pkg/dataingestionframework/data/dif_types.go

```diff
@@ -0,0 +1,155 @@
+package data
+
+import (
+	"fmt"
+
+	set "github.com/deckarep/golang-set"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+)
+
+// USING the common DIF Data
+var DIFEntityType = map[proto.EntityDTO_EntityType]string{
```

Comment:
DIFEntityType and DIFMetricType maps are needed by appMetric like monitoring sources, these should move to a different file.
Also DIFHostType belongs to the turboDif probe or at the least in the file above